app/webpacker/css/admin/v2/components/tables.scss

```diff
@@ -43,8 +47,9 @@ table tbody tr {
   }
 }
 
-table th.actions,
-table td.actions {
+table thead tr th.actions,
+table tbody tr td.actions {
+  background-color: $v2-body-bg !important;
```

Comment:
Doe we need `!important` only in the transition or is that here to stay? I can be difficult to override. Wondering if a more specific selector would work but you thought about that already, I guess.

Reply:
Normally, I don't use that `!important` keyword. It's a nightmare for me, and avoid the use of a more specific selector. I used it because it was previously used, unfortunately:

https://github.com/openfoodfoundation/openfoodnetwork/blob/c54942d1ce4915a516038fb4abb3b7db7a2490bf/app/webpacker/css/admin/components/actions.scss#L28-L30